style.css

```diff
@@ -1099,6 +1099,7 @@ table.conjunctionexamples th {
   background-color: #ddffcc;
 }
 
+.linkme:hover, .linkme:focus { text-decoration: none; }

```

Comment:
This removes the tiny little dash that appears in the top-right when you hover here:

![image](https://user-images.githubusercontent.com/16232127/134934761-3e9f7a59-04a9-4e67-8ad0-b97159be8610.png)